src/Component/DataInput/DataLoader/DataLoader.tsx

```diff
@@ -61,11 +61,14 @@ interface DataLoaderDefaultProps {
 export interface DataLoaderProps extends Partial<DataLoaderDefaultProps> {
   /** List of data parsers to use */
   parsers: DataParser[];
+  /** Properties that get passed to the UploadButton. */
+  uploadButtonProps?: Omit<UploadButtonProps, 'name' | 'action'>;
```

Comment:
```suggestion
  uploadButtonProps?: UploadButtonProps;
```

Should be sufficient, right?